### ISSUE TITLE
@viewport test failing

### ISSUE THREAD

**Issue by webfella** [CSSLint/csslint#565]
I've just created a new fork with a fresh install of dependencies, and on the first run before editing any file's I'm getting the following error (OSX Yosemite):

```1) testSuite_1421019279354_1 > General Tests for all Rules > General Tests for order-alphabetical > Using @viewport should not result in an error : Values should be equal.
Expected: 0 (number)
Actual: 1 (number)
Stack trace:
Unavailable.```

I've tried tracking it down myself, but can't seem to find anything as logging out the messages for that test show the expected value.

**Comment by webfella** [comment]
Found a PR up that resolves this issue #562

**Comment by nschonni** [comment]
Thanks, I finally resolved the issue in #545
